step-plans/step-plans-core/src/main/java/step/core/artefacts/reports/aggregated/AggregatedReportView.java

```diff
@@ -0,0 +1,39 @@
+package step.core.artefacts.reports.aggregated;
+
+import step.core.artefacts.AbstractArtefact;
+
+import java.util.List;
+import java.util.Map;
+
+public class AggregatedReportView {
+
+    public final AbstractArtefact artefact;
```

Comment:
Yay, immutable classes without getters and setters! (I'm serious, I like this). No need to change anything here.